README.md

```diff
@@ -1,4 +1,4 @@
-# ```libdocker```: an SDK for the Docker Engine API in C
+# ```libdocker```: an SDK for the Docker Engine API in C ( Hence removed the compatibility with cpp )
```

Comment:
What's the reason for removing compatibility with `cpp`? Would prefer to keep in unless there's a good reason.

Reply:
The Docker website cites libdocker as an unofficial C SDK. For C++, they have quoted some other library. So my reason for the removal of the compatibility was to strictly stick with the mentioned purpose, i.e. C SDK.